### PR TITLE
add test on comparison preset

### PR DESCRIPTION
**Pull request by moumoutte**

### CODE REVIEW

File: demo/tests/test_forms.py [hunk 1]
```diff
@@ -776,3 +776,27 @@ def test_from_json_raised_error(self):
...
+class TestInnnerPresets(TestCase):
+
+    def test_comparison(self):
+        class TestPresets(FormidableForm):
+            left = fields.IntegerField()
+            right = fields.IntegerField()
+
+        formidable = TestPresets.to_formidable(label='presets')
+
+        preset = Preset.objects.create(form=formidable, slug='confirmation')
```
Comment (brunobord) [inline demo/tests/test_forms.py]: shouldn't this slug be "comparison"?

File: demo/tests/test_forms.py [hunk 2]
```diff
@@ -776,3 +776,27 @@ def test_from_json_raised_error(self):
...
+
+        form_class = formidable.get_django_form_class()
+        form = form_class(data={'left': 42, 'right': 42})
+        self.assertTrue(form.is_valid())
+        form = form_class(data={'left': 42, 'right': "42"})
+        self.assertTrue(form.is_valid())
+        form = form_class(data={'left': 42, 'right': 21})
+        self.assertFalse(form.is_valid())
+        form = form_class(data={'left': 42, 'right': "21"})
+        self.assertFalse(form.is_valid())
```
Comment (wo0dyn) [inline demo/tests/test_forms.py]: What about adding the following assertion?

```py
form = form_class(data={'left': 42, 'right': 42.0})
self.assertFalse(form.is_valid())
```

File: demo/tests/test_forms.py (hunk 1 above)
Comment (wo0dyn) [inline demo/tests/test_forms.py]: +1 for "comparison" indeed.

**Comment by moumoutte** [comment]
I think you can close this. @mgu already provides a fix.

**Pull request closed without merging**